writers/batchwriter/batchwriter.go

```diff
@@ -6,7 +6,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/apache/arrow/go/v16/arrow/util"
+	"github.com/cloudquery/plugin-sdk/v4/batch"
```

Comment:
Let's do `internal/batch` so we don't have to worry about maintaining API compatibility

Reply:
a6e73a10440d2f50ffca1e945f69464a19a2b39f